mkdocs.yml

```
@@ -104,6 +104,7 @@ nav:
           - RStudio: ide_integration/r_studio.md
   - Tutorials:
       - ROS 2: tutorials/ros2.md
+      - Python: tutorials/python.md
```

```suggestion
      - Python: tutorials/python.md
      - ROS 2: tutorials/ros2.md
```
Definitely more important!